packages/cli/src/index.ts

```diff
@@ -179,6 +180,11 @@ yargs
             description: 'Maximum execution time in seconds.',
             type: 'number',
           },
+          'ignore-deployment-failures': {
```

Comment:
I think this should be continue-on-error or something along those likes ([github uses continue-on-error](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error))

Reply:
Maybe `continue-on-deployment-error` will be better?

Reply:
We use the term `deploy` and not `deployment` universally. I'm a stickler for sticking to domain language so I think we shouldn't use the word `deployment` whenever we can avoid it.

Reply:
Updated to `continue-on-deploy-failures`, thanks! @adamaltman